binary-tarball-tests/pxc/run.sh

```diff
@@ -1,18 +1,30 @@
 #!/usr/bin/env bash
 export PATH=${HOME}/.local/bin:${PATH}
 
+PXC_MAJOR_VERSION="$(echo ${PXC_VERSION}|cut -d'.' -f1,2)"
+
 echo "Installing dependencies..."
 if [ -f /etc/redhat-release ]; then
   sudo yum install -y libaio numactl openssl socat lsof
   # below needed for 5.6 mysql_install_db
   sudo yum install -y perl-Data-Dumper
   if [ $(grep -c "release 6" /etc/redhat-release) -eq 1 ]; then
     sudo yum install -y epel-release centos-release-scl
+<<<<<<< HEAD
+=======
+    if [ "${PXC_MAJOR_VERSION}" = "5.7" ]; then
+      sudo yum install -y libev
+    fi
+>>>>>>> abb1921b (Install xtrabackup 2.4 to test PXC 5.7)
```

Comment:
this looks like an unresolved conflict

Reply:
thanks for spotting that! some rebase that went wrong, fixing it now :)